docs/contributor-guide/developer-guide/deploy-system/prerequisits.md

```diff
@@ -3,12 +3,256 @@
+    - `APP_PREFIX="<your_dev_account_name>"` (Note: App prefix allows you to have more than one deployment in the same account. It can be set to whatever you want.)
+    - `SSL_CERTIFICATE_ARN="<arn_of_the_SSL_cert>"` (Find the ARN by clicking into the Certificate ID on the [Certificates page](https://us-east-1.console.aws.amazon.com/acm/home?region=us-east-1#/certificates/list) in AWS.)
+### [cloud-deploy-infra](https://github.com/civiform/cloud-deploy-infra)
+1. Clone [cloud-deploy-infra](https://github.com/civiform/cloud-deploy-infra) onto your local machine.
+2. Set your local copy of `civiform-deploy` to track your local copy of `cloud-deploy-infra`.
+    - In your local copy of `civiform-deploy`, update [this line](https://github.com/civiform/civiform-deploy/blob/4624131e74ff3a417d070f0a34da55744f26c347/bin/lib/checkout.sh#L86) in the `checkout::initialize` function in `checkout.sh` to point to your local repository instead of the remote origin.
+    - Eg. change 
+    ```
+    git remote add origin http://github.com/civiform/cloud-deploy-infra
+    ```
+    to 
+    ```
+    git remote add origin <path_to_local_repo>/cloud-deploy-infra
+    ```
+
+## Setup AWS Nuke
+AWS Nuke is a package that makes it easy to remove all resources from an AWS account. You will want to run this at the end of each working session (at least at the end of each day so we don't leave resources running overnight) and anytime you need to cleanup your AWS account to test new changes. It is destructive, so it's very important to set up your config file correctly to only clear out resources from your dev account. Check out the [README for the `aws-nuke` repository](https://github.com/rebuy-de/aws-nuke) for more information about the inner workings of `aws-nuke`.
+1. Download the correct binary for your machine's architecture/OS from the [aws-nuke repo](https://github.com/rebuy-de/aws-nuke/releases). Or on mac run `brew install aws-nuke`
+2. If you downloaded the binary, unpack the tarball, rename the binary to `aws-nuke` and put it somewhere in your shell's path.
+3. Run `aws-nuke -h` to make sure it is installed correctly.
+4. Create a `nuke.yaml` file and copy the [`nuke.yaml` file used for end to end](https://github.com/civiform/cloud-deploy-infra/blob/main/e2e-test/nuke.yaml) tests this file.
+5. Update the file with the correct blocklist and accounts. This will look different depending on the AWS root account you are using, but should end up looking something like:
+    ```
+    regions:
+    - us-east-1
+    - global
+
+    account-blocklist:
+    - "<account_number_1_to_blocklist>" # Description of account number 1
+    - ... additional accounts to blocklist if have them ...
+
+    accounts:
+    "<your_new_account_number>":
+        filters:
+        ACMCertificate:
+            - property: "DomainName"
+            value: "<your_user_name>.civiform.dev"
+        IAMRole:
+            - "OrganizationAccountAccessRole"
```

Comment:
My default `nuke.yaml` looks a little different.  Should we be changing it to conform to what you have here, or are these other defaults okay to leave in?

```
      filters:
        ACMCertificate:
          - property: "DomainName"
            value: "cloud-deploy-infra-tests-0.civiform.dev"
        IAMOpenIDConnectProvider:
          - type: contains
            value: "oidc-provider/token.actions.githubusercontent.com"
        IAMRole:
          - "OrganizationAccountAccessRole"
          - "e2e-test-runner"
        IAMRolePolicyAttachment:
          - "OrganizationAccountAccessRole -> AdministratorAccess"
          - "e2e-test-runner -> AdministratorAccess"
```